dev/benchmarks/complex_layout/android/settings.gradle

```diff
@@ -8,8 +8,6 @@
 
 include ':app'
 
-enableFeaturePreview('ONE_LOCKFILE_PER_PROJECT')
```

Comment:
Why was this removed? I

Reply:
This was removed in the original pr because one lockfile per project was an experimental feature that was promoted to stable.